regression/scripts/electric_performance/electric_payload_range.py

```diff
@@ -63,12 +63,13 @@ def mission_setup(vehicle, analyses):
 
     # base segment
     base_segment = Segments.Segment()
-    ones_row = base_segment.state.ones_row
-    base_segment.state.numerics.number_control_points = 4
-    base_segment.process.iterate.conditions.stability = SUAVE.Methods.skip
-    base_segment.process.finalize.post_process.stability = SUAVE.Methods.skip
+    ones_row                                                 = base_segment.state.ones_row    
+    base_segment.state.numerics.number_control_points        = 4
+    base_segment.process.iterate.conditions.stability        = SUAVE.Methods.skip
+    base_segment.process.finalize.post_process.stability     = SUAVE.Methods.skip    
+    base_segment.process.iterate.conditions.planet_position  = SUAVE.Methods.skip    
     base_segment.process.iterate.initials.initialize_battery = SUAVE.Methods.Missions.Segments.Common.Energy.initialize_battery
-    base_segment.process.iterate.conditions.planet_position = SUAVE.Methods.skip
+    base_segment.battery_discharge                           = True 
```

Comment:
This new nomenclature could get confusing. Is there another way of setting this?

Reply:
Found a nice workaround. I addded the discharge flag to the segment defaults for the charge and discharge segments. If the flag is not found (for example for typical flight segments), the flag is defaulted in the initialization to true. That way the user never sees/ does not get confused

Reply:
I was very worried about this so thanks for pushing me to find an improvement